### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by gsmalik030**
In this PR:

I added styling for my project.

### CODE REVIEW

File: src/components/styles/book.css [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+.bookSection {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  margin: 2rem 5%;
+  width: 90%;
+  background-color: white;
+  border: solid 1px #e8e8e8;
+}
```
Comment (roshan-bajgain) [inline src/components/styles/book.css]: - [ ] According to the project requirement styling is exactly the same as provided on the Zeplin. You need to make a small change on your styling that is color for `book and categories` on the navbar.
- [ ] Kindly updated the color of `Updated progress` and input for the button. See the image below for reference
- [ ] Also make sure to add the same logo that is on the right. For this, you can use the credentials from project requirement to login
### Your design
his


### Original design
figma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @gsmalik030, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Professional README file. ✔️

## Required Changes ♻️
- [ ] [Optional] Please write your commit message in impressive form and the first letter should be capitalized. Keep in mind for the next milestone
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navbar.jsx [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
-    <ul>
-      <li><Link to="/">Books</Link></li>
-      <li><Link to="/categories">Categories</Link></li>
-    </ul>
+  <nav className='navBar'>
+    <div className="navContainer">
+      <p className='logo'>Bookstore CMS</p>
+      <ul className='navLinks'>
+        <li><Link to="/">Books</Link></li>
+        <li><Link to="/categories">Categories</Link></li>
```
Comment (mahbubul14) [inline src/components/Navbar.jsx]: - [ ] The navbar menu doesn't match the Figma design. Kindly match the font color. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
your1 | figma1

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @gsmalik030,

Great work on making the changes requested by the previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "Fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by julie-ify — APPROVED** [review]
Hi @gsmalik030,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**